Targets/Extensions/watchOS-Extension.xcconfig

```diff
@@ -0,0 +1,19 @@
+//
+// watchOS-Extension.xcconfig
+//
+// Copyright (c) 2016 Twitter. All rights reserved.
+//
+
+#include "ExtensionTarget.xcconfig"
+
+SUPPORTED_PLATFORMS = watchsimulator watchos
+
+SDKROOT = watchos
+
+CODE_SIGN_IDENTITY = iPhone Developer
```

Comment:
Actually, `Target.xcconfig` already includes it, so I will remove this setting from here altogether